src/utils/network/session-core.js

```diff
@@ -356,9 +353,13 @@ export const deleteDocumentContainer = async (session, documentUrl) => {
  * documents being uploaded by authorized users
  */
 
-export const createDocumentContainer = async (session) => {
-  const userContainerUrl = getContainerUrl(session, 'Documents', 'self-fetch');
-  await createContainerAt(userContainerUrl, { fetch: session.fetch });
+export const createDocumentContainer = async (session, podUrl) => {
+  const userContainerUrl = `${podUrl}Documents/`;
+  try {
+    await createContainerAt(userContainerUrl, { fetch: session.fetch });
```

Comment:
This line can throw an error if the container already exists (although this behavior seems to vary from server to server; node solid server seems to return 200 if the container already exists). So let's handle the error appropriately.

Reply:
Think this is the strategy for the refactoring in PR #201. Could leave the refactoring to there.